layers/build-krd/dracut/soci/soci-settled.sh

```diff
@@ -192,6 +194,29 @@ soci_udev_settled() {
             return 1
         fi
 
+        action=$(<${rootd}/mos-action)
```

Comment:
You're effectively trusting the signer of the manifest to tell you whether or not to extend PCR7 (and thus stop them from having access to the protected TPM values).

it feels to me like that isn't right.

Reply:
I opened an issue to discuss hte better way to do this.